xml/docu_styleguide.language.xml

```diff
@@ -371,14 +373,13 @@
   <title>Tense</title>
   <para>
    Use the simple present tense. Apply the simple present tense even to
-   sentences with <quote>if</quote> or <quote>when</quote> clauses:
+   sentences with <quote>if</quote> or <quote>when</quote> clauses and to
+   prerequisites of an action:</para>
+  <para>
    <quote>If this happens, go there.</quote>
   </para>
   <para>
-   Prerequisites of an action should be expressed in the present tense as
-   well: <quote>Glibc is installed.</quote> In some cases, no verb is
-   necessary before the prerequisite of an action: <quote>a 1&nbsp;GHz
-   processor or better.</quote>
+   <quote>Glibc is installed.</quote>
```

Comment:
I like the idea, I also think it makes sense to remove the barely relevant example of a prereq without a verb.

However, two paras that each only have a single, quoted example in them is a bit weird and would look better as a list. It's also a bit less clear now because the two examples now aren't clearly associated with the categories (i.e. *if clause* or *prereq*) that they fit into.

Reply:
@sknorr : I'm not sure I understand your comment. Could you create a suggestion, please?

Reply:
Closing unchanged.